tests/api2/test_charts_schema.py

```diff
@@ -49,5 +49,5 @@ def test_text_schema_max_length(request):
                 'values': {'testtext': secrets.token_hex(2 * 1024 * 1024 // 2)},
             }):
                 assert ve.value.errors[0].errmsg == (
-                    'values.testtext: Value greater than 1048576 not allowed'
+                    'values.testtext: The value may not be longer than 1048576 characters'
```

Comment:
@sonicaj this message here comes from the middleware, right? Thought it might as well come from catalog_validation, but wasn't able to find that code there.

Reply:
@themylogin it indeed comes from middleware